File: nicolaspuebla_proyecto_final/src/main/java/com/example/nicolaspuebla_proyecto_final/model/dto/MemberListElement.java
package com.example.nicolaspuebla_proyecto_final.model.dto;

import com.example.nicolaspuebla_proyecto_final.model.dataModels.MobileUser;

public class MemberListElement {
    private String rolType;
    private MobileUser user;

    public MemberListElement() {}

    public MemberListElement(String rolType, MobileUser user) {
        this.rolType = rolType;
        this.user = user;
    }

    public String getRolType() {
        return rolType;
    }

    public void setRolType(String rolType) {
        this.rolType = rolType;
    }

    public MobileUser getUser() {
        return user;
    }

    public void setUser(MobileUser user) {
        this.user = user;
    }
}

File: nicolaspuebla_proyecto_final_android/app/src/main/java/com/example/nicolaspuebla_proyecto_final_android/data/model/dto/MemberListElement.kt
package com.example.nicolaspuebla_proyecto_final_android.data.model.dto

import com.example.nicolaspuebla_proyecto_final_android.data.model.dataClases.MobileUser

/**
 * Clase que forma parte de una lista de miembros de un equipo junto con su rol en el mismo.
 */
data class MemberListElement (
    val rolType: String,
    val user: MobileUser
)